db/migrate/20160915144154_add_color_to_tags.rb

```diff
@@ -0,0 +1,5 @@
+class AddColorToTags < ActiveRecord::Migration
+  def change
+    add_column :tags, :color, :string, default: "#f50"
```

Comment:
hmmmmmm I'm super on the fence about this..... but.... I'm willing to give it a shot..... 😉